app/policies/follow_request_policy.rb

```diff
@@ -0,0 +1,23 @@
+class FollowRequestPolicy < ApplicationPolicy
+  attr_reader :user, :follow_request
+
+  def initialize( user, follow_request )
+    @user = user
+    @follow_request = follow_request
+  end
+
+  def create?
+    true
+  end
+
+  def destroy?
+    true
+  end
+
+  def update?
+    true
```

Comment:
Could be more specific here, only allow sender or recipient to update a FollowRequest